src/TodoForm.jsx

```diff
@@ -1,10 +1,18 @@
 function TodoForm() {
   return (
-    <>
+    <form /*onSubmit={(event) => onsubmit(event)} this is a button submit action*/
+    >
       <label htmlFor="todoTitle">Todo</label>
-      <input id="todoTitle" />
-      <button>Add Todo</button>
-    </>
+      <input id="todoTitle" placeholder="Please add your todo's" />
```

Comment:
Nice use of `placeholder`!

Reply:
It's good to be mindful of the assignment requirements but something like `placeholder` doesn't necessarily jump beyond the lesson, it doesn't clutter your code, and it brings more clarity to your code. I think this is fine to implement even though not explicitly instructed.